Description: Find gtest source files.
 Upstream defaults to downloading a gtest tarball and using that for build.
 Instead, we use the libgtest-dev package.  We need to tell the build system
 how to find these files.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-03-18

--- a/src/test/gtestInclude.cpp
+++ b/src/test/gtestInclude.cpp
@@ -7,4 +7,4 @@
 
 // the .. goes back from the include/ directory of gtest so we can
 // enter the src directory.
-#include <../src/gtest-all.cc>
+#include <src/gtest-all.cc>
--- a/configure.ac
+++ b/configure.ac
@@ -57,7 +57,7 @@
   [AC_MSG_ERROR([invalid value $with_gtest for with_gtest.])]
 )
 AS_IF([test "x$with_gtest" == "xyes"],
-  [GTEST_CFLAGS="-I`cd $GTEST_PATH/gtest/include; echo $PWD` -I`cd $GTEST_PATH/gtest/; echo $PWD`"]);
+  [GTEST_CFLAGS="-I/usr/include/gtest -I`cd $GTEST_PATH/gtest/; echo $PWD`"]);
 AM_CONDITIONAL(with_gtest, test "x$with_gtest" == "xyes")
 
 DEPS_CFLAGS="$GTEST_CFLAGS"
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,7 +3,7 @@
 ACLOCAL_AMFLAGS = -I build/autotools/m4
 
 # Options passed to the C PreProcessor (CPP), NOT the C Plus Plus compiler.
-AM_CPPFLAGS = -I${top_srcdir}/
+AM_CPPFLAGS = -I${top_srcdir}/  $(DEPS_CFLAGS)
 
 # tell Libtool what the name of the library is.
 lib_LTLIBRARIES = libmemtailor.la
